fares-data-build-tool/create-data
# feat: temporary tables for reference data import

# Description

-   Currently we clear out the tables at the beginning of every day - instead of doing this, and having no data if the import fails, we are creating some new temporary tables to hold the data as the import happens.

# Testing instructions

-   Run the new lambda and the reference data lambdas afterwards, to ensure they work in sequence.

# Type of change
